osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs

```diff
@@ -73,12 +77,44 @@ private void load()
                     Origin = Anchor.TopCentre,
                     Current = tabletHandler.Enabled
                 },
-                noTabletMessage = new OsuSpriteText
+                noTabletMessage = new FillFlowContainer
                 {
-                    Text = TabletSettingsStrings.NoTabletDetected,
-                    Anchor = Anchor.TopCentre,
-                    Origin = Anchor.TopCentre,
-                    Padding = new MarginPadding { Horizontal = SettingsPanel.CONTENT_MARGINS }
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    Direction = FillDirection.Vertical,
+                    Padding = new MarginPadding { Horizontal = SettingsPanel.CONTENT_MARGINS },
+                    Spacing = new Vector2(5f),
+                    Children = new Drawable[]
+                    {
+                        new OsuSpriteText
+                        {
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            Text = TabletSettingsStrings.NoTabletDetected,
+                        },
+                        new LinkFlowContainer
+                        {
+                            TextAnchor = Anchor.TopCentre,
+                            Anchor = Anchor.TopCentre,
+                            Origin = Anchor.TopCentre,
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                        }.With(t =>
+                        {
+                            if (RuntimeInfo.OS == RuntimeInfo.Platform.Windows || RuntimeInfo.OS == RuntimeInfo.Platform.Linux)
+                            {
+                                t.NewLine();
+                                t.AddText("If your tablet is not getting detected properly,", s => s.Colour = colours.Yellow);
+                                t.NewParagraph();
+                                t.AddText("read ", s => s.Colour = colours.Yellow);
+                                t.AddLink("the driver's FAQ", LinkAction.External, RuntimeInfo.OS == RuntimeInfo.Platform.Windows
+                                    ? @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ"
+                                    : @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Linux-FAQ");
+
+                                t.AddText(" to troubleshoot the problem further.", s => s.Colour = colours.Yellow);
```

Comment:
```suggestion
                                t.AddText("If your tablet is not detected, please read ", s => s.Colour = colours.Yellow);
                                t.AddLink("this FAQ", LinkAction.External, RuntimeInfo.OS == RuntimeInfo.Platform.Windows
                                    ? @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Windows-FAQ"
                                    : @"https://github.com/OpenTabletDriver/OpenTabletDriver/wiki/Linux-FAQ");
                                t.AddText(" for troubleshooting steps.", s => s.Colour = colours.Yellow);
```

It's not really a driver (misleading, I know), so we shouldn't be calling it that.